azurerm/resource_arm_app_service.go

```diff
@@ -214,6 +214,25 @@ func resourceArmAppService() *schema.Resource {
 				Computed: true,
 			},
 
+			"ip_restriction": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"ip_address": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"subnet_mask": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Default:  "255.255.255.255",
```

Comment:
context from the Azure Portal: `Specify an IP address range by providing a subnet mask (i.e. 255.255.255.0). If no mask is provided 255.255.255.255 is assumed.`